presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -153,6 +153,23 @@ Mathematical Functions
     array ``bins``. The ``bins`` parameter must be an array of doubles and is
     assumed to be in sorted ascending order.
 
+Statistical Functions
+-----------------------
+
+.. function:: wilson_interval_lower(pos, neg[, z]) -> double
```

Comment:
I don't think we should have a version that doesn't take the z-score. The choice of 1.96 (or 95%) seems very arbitrary.

It would also be good to have a function that converts a confidence level to a z-score for ease of use (or have the function take a confidence level instead).